code/modules/power/smes_construction.dm

```diff
@@ -316,7 +339,23 @@
 	// - Hatch is open, so we can modify the SMES
 	// - No action was taken in parent function (terminal de/construction atm).
 	if (..())
-
+		if(W.iswelder())
+			if(health == initial(health))
+				to_chat(user, SPAN_WARNING("\The [src] is already repaired."))
```

Comment:
Probably better to do something like "\The [src] isn't in need of repairs." or "\The [src] doesn't need repairs." Otherwise it's kinda weird to see this message from an SMES that was never damaged.